### PR TITLE
Expose two more imgui functions

### PR DESCRIPTION
**Pull request by fdejaeger**

### CODE REVIEW

File: imgui.go [hunk 1]
```diff
@@ -393,6 +393,21 @@ func SliderFloat(label string, value *float32, min, max float32) bool {
 	return SliderFloatV(label, value, min, max, "%.3f", 1.0)
 }
 
+// SliderFloat3V creates slider for a 3D vector
```
Comment (dertseha) [inline imgui.go]: Sentence misses a period :)

File: imgui.go [hunk 2]
```diff
@@ -393,6 +393,21 @@ func SliderFloat(label string, value *float32, min, max float32) bool {
...
+func SliderFloat3V(label string, values *[3]float32, min, max float32, format string, power float32) bool {
+	labelArg, labelFin := wrapString(label)
+	defer labelFin()
+	formatArg, formatFin := wrapString(format)
+	defer formatFin()
+	cvalues := (*C.float)(&values[0])
+	return C.iggSliderFloatN(labelArg, cvalues, 3, C.float(min), C.float(max), formatArg, C.float(power)) != 0
+}
+
+// SliderFloat3 creates slider for a 3D vector
```
Comment (dertseha) [inline imgui.go]: Please have it state `SliderFloat3 calls SliderFloat3(label, values, min, max, "%.3f", 1.0).` to indicate the used "defaults".

File: imgui.go [hunk 3]
```diff
@@ -638,3 +653,7 @@ func IsItemHoveredV(flags int) bool {
 func IsItemHovered() bool {
 	return IsItemHoveredV(HoveredFlagsDefault)
 }
+
+func IsKeyPressed(key int) bool {
```
Comment (dertseha) [inline imgui.go]: Misses documentation, please add - especially that it is about the user-key-index, not the imgui-internal one.

File: imgui.go (hunk 1 above)
Comment (fdejaeger) [reply imgui.go]: done.

**Comment by dertseha** [comment]
Thank you for the update!
I'm waiting for the other pull request to go through, then I'll make a release.

**Pull request merged**